lib/tasks/export_tables_as_csv_to_big_query.rake

```diff
@@ -1,10 +0,0 @@
-namespace :tables_as_csv do
-  desc 'Exports postgres tables to Big Query'
-  namespace :to_big_query do
-    task export: :environment do
-      # It doesn't use CSV anymore, but I'm not changing the task name for now to avoid having to change more of the
-      # infrasctructure than strictly necessary
-      ExportTablesToBigQueryJob.perform_later
```

Comment:
I thought this was how Steven got his data?

Reply:
We don't use the rake tasks

Reply:
When we were on AWS, the way to do cron jobs was to create rake tasks and set up some kind of cronjobs that invoked them. When we moved to GOV.UK PaaS, we now have Sidekiq Cron, a gem that invokes directly the jobs, no need to create rake tasks. You could argue that when you have to run them manually, you can invoke them, but calling the job from within the console is not the same amount of typing.